languages/translations/sv.json

```diff
@@ -165,24 +165,24 @@
                 "destination": "Körfältsbyte åt {modifier} mot {destination}"
```

Comment:
Can we change these too or would the definite form adjective make the noun clash?
Something like {modifier.definiteForm} vänster/vänstra would be convenient.

Reply:
I would like to change these too. But I'm not aware if it's even possible. Since the modifier needs to be `vänster` for other instructions. Guess there is no way to extend the modifier to have a form too? But this should be changed since it's an instruction without a verb.